nixos/modules/services/networking/nftables.nix

```diff
@@ -279,7 +279,7 @@ in
             ''}
           '';
           checkPhase = lib.optionalString cfg.checkRuleset ''
-            cp $out ruleset.conf
+            cp ${if lib.isStorePath cfg.rulesetFile then cfg.rulesetFile else "$out"} ruleset.conf
```

Comment:
I reviewed this PR again before merging it and found that changes should be made.

> When `rulesetFile` is set, `ruleset.conf` is just a file that includes that.

This is no longer true since https://github.com/NixOS/nixpkgs/pull/207758.

I think now we should `cp $out ruleset.conf`, copy rulesetFile and then make `include "${cfg.rulesetFile}"` in ruleset.conf point to our copied rulesetFile.